tests/unit/options.js

```diff
@@ -1443,9 +1443,16 @@ exports.browser = function (test) {
     .addError(25, "'DocumentFragment' is not defined.")
     .addError(26, "'Range' is not defined.")
     .addError(27, "'Text' is not defined.")
+    .addError(30, "'console' is not defined.")
+    .addError(31, "'alert' is not defined.")
     .test(src, {es3: true, undef: true });
 
-  TestRun(test).test(src, {es3: true, browser: true, undef: true });
+  TestRun(test)
+    .addError(30, "'console' is not defined.")
+    .addError(31, "'alert' is not defined.")
+    .test(src, {es3: true, browser: true, undef: true });
+
+  TestRun(test).test(src, {es3: true, browser: true, devel: true, undef: true });
```

Comment:
The intention of the above changes is not breaking but ensuring that jshint keeps current behavior. Please let me explain.

Currently, jshint reports errors for following

```
/* jshint browser:true, undef:true */
console.log("hello");
alert("hola");
```

but it doesn't report errors

```
/* jshint browser:true, undef:true, devel:true */
console.log("hello");
alert("hola");
```
as explained in http://jshint.com/docs/options/#devel

I would like to keep this behavior (or feature). Therefore I added  lines to test.

First I added console and alert to the test fixture.
https://github.com/jshint/jshint/pull/2169/files#diff-1d39308f00a655c961bd6636ed22e255R28

```
// check for devel
console.log("test");
alert("test");
```
Then I added code that ensures that jshint still reports errors for console, alert without devel option.
```
   .addError(30, "'console' is not defined.")
   .addError(31, "'alert' is not defined.")
     .test(src, {es3: true, undef: true });

 TestRun(test)
   .addError(30, "'console' is not defined.")
   .addError(31, "'alert' is not defined.")
   .test(src, {es3: true, browser: true, undef: true });
 ```

Finally I added codes that ensures that devel option still works correctly.

```
TestRun(test).test(src, {es3: true, browser: true, devel: true, undef: true });
```

If it is better that I create a separate test for `devel` option, please let me know.